pkg/util/errors/errors.go

```diff
@@ -29,10 +29,16 @@ type Aggregate interface {
 // is itself an implementation of the error interface.  If the slice is empty,
 // this returns nil.
 func NewAggregate(errlist []error) Aggregate {
-	if len(errlist) == 0 {
+	errs := []error{}
+	for _, err := range errlist {
+		if err != nil {
```

Comment:
I disagree, I think, because it doesn't feel Go-like.  Can you show me where this would make a significant difference to readability?